packages/salesforcedx-vscode-core/src/services/sdr/componentSetUtils.ts

```diff
@@ -0,0 +1,26 @@
+import { ConfigUtil } from '@salesforce/salesforcedx-utils-vscode/src';
```

Comment:
The above shouldn't include the src part of the path.  This is also why your int tests failed.